src/app/features/home/capture-tab/capture-tab.component.ts

```diff
@@ -248,6 +251,10 @@ export class CaptureTabComponent implements OnInit {
       .subscribe();
   }
 
+  async openFaq() {
```

Comment:
Please solve or set ignore the lint failure `Expected 'this' to be used by class async method 'openFaq'`